README.md

```diff
@@ -1,16 +1,56 @@
-# Title
+# What Makes an Effective Programmer?
 
-## Learning Goals
+What do the best programmers do that the average programmers don't?
 
--SWBAT 1
--SWBAT 2
+_Do they write more code faster?_
 
-## Introduction
+Can we say that because person A wrote 10,000 lines and person B wrote 500
+that A is outperforming B? No, we can't. In the world of software development
+500 lines of code can do the work of 10,000 and it's much easier to work on the
+500 lines than 10,000.
 
-## SWBAT 1
+_Do they get code to pass tests more quickly?_
 
-## SWBAT 2
+Sometimes! But since programmers write the tests as well as their implementation
+code—the code that satisfies the tests—they can only test what they themselves
+know about. An inexperienced programmer will write low-value tests that low-
+value code can pass quickly, and sleep soundly not knowing how blind they were.
+The measure of code is not passing already written tests, but creating the
+potential to expand on the code's behavior in the future.
 
-## Conclusion
+The difference is _process_.
 
-## Resources
+Effective programmers take the time to _really_ understand the problem
+they're trying to solve. Once they understand it, they make focused, decided
+steps with _mindfulness_ to implement the solution. **Learning to program is the
+art of solving problems.** It's the main muscle that you are constantly
+developing.
+
+Let's dive in a bit deeper here. What is problem solving?
+
+## Problem Solving in the Abstract
+
+Think about some of the problems people have on a regular basis. If you're a
+soccer player, getting past the defense to score a goal is a problem, because
+there are people blocking you from kicking the ball into the net. If you're
+hungry, making a peanut butter and jelly sandwich is a problem, because you need
+to find the ingredients and assemble them in the correct order. Whether we call
+it a "play" on the soccer field or a "recipe" in the kitchen, essentially what
+you're doing is following a process to create _another_ process &mdash; a
+process that will solve the problem.
+
```

Comment:
just a suggestion, might not be necessary, but maybe carry the analogies further - good chefs don't need to read a recipe, but they've replaced the need for it with their own internal process...  similarly, good programmers internalize processes for approaching..

Reply:
or maybe the content specific to programming should be in its own section, since the previous paragraph is about problem solving in the abstract. 

There may be room then to also talk a bit more explicitly about what problem solving we do as programmers in this course and in general.